src/views/index.ejs

```diff
@@ -78,7 +78,7 @@
                   <th scope="row"><%=clip.click_count%></th>
                   <td><%=clip.long_url%></td>
                   <td>
-                    <a class="trimmed" target="_blank" href="https://<%=clip.long_url%>"
+                    <a class="trimmed" target="_blank" href="<%=clip.long_url%>"
```

Comment:
what happens if the user doesn't add https:// when trimming?

Reply:
then the issue is from the validation

Reply:
if the guys handling the validation could require the user to add the Http. this is why we should have used an npm validation package. it would have saved us this error.  and with this much pull request, not sure if someone has already made the corrections.